src/ImageProcessor/Imaging/ResizeLayer.cs

```diff
@@ -102,67 +103,49 @@ public class ResizeLayer
         /// <summary>
         /// Gets or sets the center coordinates.
         /// </summary>
-        public float[] CenterCoordinates { get; set; }
+        public PointF? CenterCoordinates { get; set; }
```

Comment:
This is a binary breaking change (changing the type of a public property on a public class) and will require all code using it to be recompiled!

Reply:
Hmmm... I doubt very much anyone would update the library without compilation.

Reply:
Not so fast! If another package uses ImageProcessor (e.g. Umbraco or your own utility package), that code won't automatically recompile and will require releasing a new version that only works on the latest ImageProcessor release.

This totally breaks backwards compatibility and will require the package maintainers to patch all supported versions. So this change can only be done when releasing ImageProcessor 3.

Reply:
I've seen these kind of breaking changes multiple times and they're quite hard to notice, as tests are always recompiled and will just use the new signatures. They all reverted the change and released a patched version:
- https://github.com/zzzprojects/html-agility-pack/issues/375
- https://github.com/umbraco/Umbraco-CMS/issues/5822

Reply:
Thing is, I really don't care. I have no obligation to any of those projects.

Reply:
The same applies to your own packages, although I see you've already updated ImageProcessor.Web in this PR to require the latest version (but I'm not sure if other packages/plugins are also affected by this).

Besides the binary breaking change, won't this also swap the Y,X notation that's currently used? That would be another functional breaking change that can't be fixed with a recompile or even simple code changes. URLs including this notation might be used/stored somewhere (hardcoded in HTML or in the database) and this might even cause wrong images to be returned from the cache (and because of browser caching can't be simply fixed by clearing all cached images)!

Reply:
If the coordinates are indeed swapped, this would also be a problem for ImageProcessor 3...

Reply:
Ah, good point on Y, X. I forgot about that.

Reply:
As the swapped coordinates are only part of ImageProcessor.Web and support for that has dropped for version 3 (at least for now), this shouldn't cause a problem.